db/migrate/20230316050000_create_yard_sales.rb

```diff
@@ -0,0 +1,15 @@
+class CreateYardSales < ActiveRecord::Migration[7.0]
+  def change
+    create_table :yard_sales do |t|
+      t.integer :user_id
+      t.integer :address_id
+      t.text :title
+      t.text :description
+      t.date :date
+      t.time :hours
+      t.integer :image_id
```

Comment:
Same question here. We should be indicating the ones that are required. Also, we may want `user_id` and `address_id`  to be foreign keys.

One question, are yard sales supposed to only have one image?

Reply:
it would be something like this right?
t.integer :user_id, foreign_key: true

Reply:
```t.references :user, null: false, foreign_key: true```
 This article may be useful to understand how that works: https://medium.com/@where_is_x/activerecord-t-references-foreign-key-and-belongs-to-e58cebc79493